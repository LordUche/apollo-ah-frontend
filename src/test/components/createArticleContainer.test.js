import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import CreateArticleContainer from '../../components/CreateArticles';

configure({ adapter: new Adapter() });

describe('<CreateArticleContainer />', () => {
  it('should render <CreateArticleContainer /> component given the props', () => {
    const container = shallow(<CreateArticleContainer />);
    expect(container).toMatchSnapshot();
  });
});
