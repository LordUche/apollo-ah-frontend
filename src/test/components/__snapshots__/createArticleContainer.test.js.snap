// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateArticleContainer /> should render <CreateArticleContainer /> component given the props 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
